code/datums/ai/robot_customer/robot_customer_controller.dm

```diff
@@ -58,3 +67,51 @@
 		attending_venue.on_get_order(pawn, order_item)
 
 
+///Called when
+/datum/ai_controller/robot_customer/proc/on_get_pulled(datum/source, mob/living/puller)
+	SIGNAL_HANDLER
+	var/mob/living/simple_animal/robot_customer/customer = pawn
+	var/datum/customer_data/customer_data = blackboard[BB_CUSTOMER_CUSTOMERINFO]
+	var/datum/venue/attending_venue = blackboard[BB_CUSTOMER_ATTENDING_VENUE]
+
+	var/obj/item/card/id/used_id = puller.get_idcard(TRUE)
```

Comment:
Named argument for TRUE.

Reply:
![image](https://user-images.githubusercontent.com/20987591/110383783-13f3f100-805d-11eb-9e52-dabf02610ffa.png)